README.md

```diff
@@ -25,3 +26,10 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+$T(n) = {C     if n \le 1,  
+3T(n/3) + O(n)     if n \gt 1}$
```

Comment:
You're still not doing a linear amount of work in addition to the recursive calls.